CF/deploy-app.sh

```diff
@@ -105,8 +110,14 @@ if [[ "$CF_SPACE" == "staging" || "$CF_SPACE" == "prod" ]]; then
 
   MEMORY_LIMIT="512M"
   INSTANCE_COUNT="3"
-  SIDEKIQ_MEMORY_LIMIT="16384M"
-  SIDEKIQ_INSTANCE_COUNT="3"
+
+  SIDEKIQ_DEFAULT_MEMORY_LIMIT="2048M"
+  SIDEKIQ_DEFAULT_INSTANCE_COUNT="1"
+  SIDEKIQ_DEFAULT_CONCURRENCY="5"
+
+  SIDEKIQ_INGEST_MEMORY_LIMIT="16384M"
+  SIDEKIQ_INGEST_INSTANCE_COUNT="2"
```

Comment:
This gives an additional 16GiB instance; have we approved this with the client?

Reply:
I've updated this to one instance of ingest for now, and increased the concurrency to 3.